tests/conf.js

```diff
@@ -18,8 +18,8 @@ const baseConfig = {
   seleniumAddress: 'http://localhost:4444/wd/hub',
   //Enabling specs as they are working.
   suites: {
-    // e2e:'e2e/**/*.js',
-    e2e: ['**/login.spec.js'],
+    e2e:'e2e/infodocs.js',
+    //e2e: ['**/login.spec.js'],
```

Comment:
Does that `login.spec.js` not work anymore? I see it's going from being executed to being commented out here. If it works, please include it. If it doesn't work and needs fixing up, please remove that line and we'll add it back in when we get to it.

Also, as you'll see in https://github.com/medic/cht-core/pull/6783 we will end up with a lot of files here. Please use the array syntax (with trailing comma) as seen here to avoid conflicts on diffs as we move forward on these.

Reply:
added that back, with a trailing comma at the end... building now